docs/Quickstart.md

```diff
@@ -0,0 +1,229 @@
+    ```
+
+## Set up solution
+
+1. **Initialize IoT Edge solution and setup Azure resources**
+
+    `iotedgedev init`
+
+    > `iotedgedev init` will create a solution and setup your Azure IoT Hub in a single command. The solution comes with a default C# module named `filtermodule`.
+
+    <details>
+    <summary>More information</summary>
+
+    1. You will see structure of current folder like below:
+
+        ```text
+            │  .env
+            │  .gitignore
+            │  deployment.debug.template.json
+            │  deployment.template.json
+            │
+            ├─.vscode
+            │      launch.json
+            │
+            └─modules
+                └─filtermodule
+                    │  .gitignore
+                    │  Dockerfile.amd64
+                    │  Dockerfile.amd64.debug
+                    │  Dockerfile.arm32v7
+                    │  Dockerfile.windows-amd64
+                    │  filtermodule.csproj
+                    │  module.json
+                    │  Program.cs
+        ```
+
+    2. Open `.env` file, you will see the `IOTHUB_CONNECTION_STRING` and `DEVICE_CONNECTION_STRING` environment variables filled correctly.
+    3. Open `deployment.template.json` file
+        1. You will see below section in the modules section:
+
+            ```json
+            "filtermodule": {
+                "version": "1.0",
+                "type": "docker",
+                "status": "running",
+                "restartPolicy": "always",
+                "settings": {
+                    "image": "${MODULES.filtermodule}",
+                    "createOptions": {}
+                }
+            }
+            ```
+
+        2. Two default routes are added:
+
+            ```json
+            "routes": {
+                "sensorTofiltermodule": "FROM /messages/modules/tempSensor/outputs/temperatureOutput INTO BrokeredEndpoint(\"/modules/filtermodule/inputs/input1\")",
+                "filtermoduleToIoTHub": "FROM /messages/modules/filtermodule/outputs/* INTO $upstream"
+            }
+            ```
+
+    4. You will see privacy statement like below:
+
+        ```text
+        Welcome to iotedgedev!
+        -------------------------
+        Telemetry
+        ---------
+        The iotedgedev collects usage data in order to improve your experience.
+        The data is anonymous and does not include commandline argument values.
+        The data is collected by Microsoft.
+        
+        You can change your telemetry settings by updating 'collect_telemetry' to 'no' in ~/.iotedgedev/setting.ini
+        ```
+
+    </details>
+
+2. **Build IoT Edge module images**
+
+    `sudo iotedgedev build`
+
+    > This step will build user modules in deployment.template.json targeting amd64 platform.
+
+    <details>
+    <summary>More information</summary>
+
+    1. You will see a "BUILD COMPLETE" for each module and no error messages in the terminal output.
```

Comment:
I think there is an mistake here? should this have been step 1, 2, 3?

Reply:
Ah, good catch. I'm making these changes to keep things consistent, but I'd like to point out markdown support numbering everything 1. and then it'll render it as a list of 1., 2., 3. ect. I prefer using real numbers though, makes it easier to keep track of when not looking at the rendered version.